### DISCUSSION TITLE
Using egs++ cavity to score dose or SPR without known cavity volume or cavity mass for each individual region

### DISCUSSION THREAD

**Discussion by karenrex1023** [nrc-cnrc/EGSnrc#1183] [Questions] [answered]
Hello,
I am trying to score dose to a region of air in a spherical ionization geometry that I have made in egs cavity. Based on how I created the geometry, the air includes several regions in egs_view. Although I have the mass for the overall, cumulative air region, I don't know the volumes or masses for the individual/specific regions of air as specified by egs_view. Is there a way I can combine all the individual regions into an overall air region in egs++ or that I can score dose to the air region without knowing the individual region masses? Below is the code I have so far:
:start scoring options:

    calculation type = dose

    :start calculation geometry:
        geometry name = chamber_in_air #spherical ion chamber
        cavity regions = 662 617 619 621 623 625 661 #regions of air within the chamber
        cavity mass = 0.00426329  #air density*volume for overall inner chamber air volume
    :stop calculation geometry:

:stop scoring options:

In addition, the documentation mentioned cavity could be used to get the stopping power ratio (SPR) as well as dose. I was wondering how the code above would change if I was trying to calculate the SPR?

**Comment by rtownson** [comment, marked as answer]
`cavity` doesn't calculate SPR, but I can see that it's easy to misread the sentence `Features available in CAVRZnrc but *not* provided by cavity are:`. 

I suggest setting `cavity mass = 1`. Then the "dose" reported will actually just be the sum of the energies deposited (almost, it's actually normalized by a constant factor for unit conversion). Then you can divide by the total mass manually to get the actual dose per fluence.

**Reply by karenrex1023** [reply]
Thank you for the suggestion!!! What is the numerical value normalization factor? I imagine if I am trying to get the actual energy deposited, I would need to divide the value given in the output file for the "dose" by the normalization factor?

**Reply by rtownson** [reply]
It's a constant, so if you want the dose deposited you can leave it since only the mass is varying between regions. If you actually want the energy deposited per fluence, just divide by `1.602e-10`.
